extensions/kind/package.json

```diff
@@ -55,14 +55,15 @@
     }
   },
   "scripts": {
-    "build": "tsc && node ./scripts/build.js",    
+    "build": "vite build && node ./scripts/build.js",
```

Comment:
Ok looks like https://github.com/containers/podman-desktop/commit/ce32dc4116a8cbfa230490a578fb1ca3225c4f76 didn't had a proper merge as it reverted some changes